src/vue-testing-library.js

```diff
@@ -115,9 +115,16 @@ async function fireEvent(...args) {
   dtlFireEvent(...args)
   await waitFor(() => {})
 }
+const changeOrInputEventCalledDirectly = (eventValue, eventKey) =>
+  eventValue && (eventKey === 'change' || eventKey === 'input')
 
 Object.keys(dtlFireEvent).forEach(key => {
   fireEvent[key] = async (...args) => {
+    if (changeOrInputEventCalledDirectly(args[1], key)) {
+      console.warn(
+        'By using "change" or "input", there may be unexpected issues. Please use "update" for a better experience.',
```

Comment:
A small suggestion to (1) write down the used event key (so people have a better feedback on where to look for) and (2) to display "fireEvent.", so that people have the exact answer right there in the warning. Any other suggestion is welcome :)

```suggestion
        `Using "fireEvent.{$key} may lead to unexpected results. Please use fireEvent.update() instead.`,
```

Reply:
This makes sense, I'll make that change, thanks!